controllers/admin/AdminThemesController.php

```diff
@@ -533,6 +540,38 @@ public function renderOptions()
                 )
             ),
         );
+		
+        if (in_array("1", array_column($this->_languages, 'is_rtl'))) {
+            $themes_list = array();
+            $allThemes = $this->theme_repository->getList();
+            foreach ($allThemes as $theme) {
+                $themeName = $theme->getName();
+                $themes_list[] = array('theme' => $themeName, 'name' => $themeName);
+            }
+            $this->fields_options['RTL'] = array(
+                'title' => sprintf($this->trans('Adaptation to Right-to-Left languages', array(), 'Admin.Design.Feature'), $this->context->shop->name),
+                'description' => $this->trans('Be careful! Please check your theme in an RTL language before generating the RTL stylesheet: your theme could be already adapted to RTL.\nOnce you click on "Adapt to RTL", the stylesheet created may delete any files you could have previously in your theme to adapt to RTL.', array(), 'Admin.Design.Help'),
```

Comment:
I think it should `...delete any file...` (singular).

I find the last sentence a bit complex to read. I suggest an alternative: Once you click on "Adapt to RTL", any RTL-specific file that you might have added to your theme might be deleted by the created stylesheet". What do you think @AlexEven ?
Also, does this mean that the new stylesheet needs non-CSS files to be deleted? Or should that be explained in more details?

Reply:
Thanks @xBorderie for the review. Yes, we can go with your suggestion! 👍